extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -859,6 +859,33 @@ public enum ResponseMode {
         @ConfigItem(defaultValue = "true")
         public boolean allowMultipleCodeFlows = true;
 
+        /**
+         * Fail with the HTTP 401 error if the state cookie is present but no state query parameter is present.
```

Comment:
> no state query parameter is present

It also impacts state when more than one state query param is present. You don't mention it because it doesn't matter - if there is multivalued state query param, you can't match it with the cookie, correct?

Reply:
Only a single value state query parameter is supported, I'll need to add a dedicated handling and failure for a multiple-valued state query

Reply:
Ok, thx